package.json

```diff
@@ -2,6 +2,17 @@
   "name": "front-robert",
   "version": "0.1.0",
   "private": true,
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,ts,tsx}": [
+      "eslint --fix",
+      "git add"
```

Comment:
I think git add is not necessary anymore and output a warning with lint-staged: `Some of your tasks use git add command. Please remove it from the config since all modifications made by tasks will be automatically added to the git commit index.`

Reply:
Ah yep indeed thanks